### PR TITLE
Dockerfilez for src added

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.14.2.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ build_root:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
 canonical_go_repository: knative.dev/eventing
 images:
+- dockerfile_path: openshift/ci-operator/source-image/Dockerfile
+  from: src
+  to: knative-eventing-src
```
Comment (mgencur) [inline ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.14.2.yaml:14]: I think there's now another way to specify the image - easier:
```
promotion:
  additional_images:
    knative-eventing-src: "src" 
```
Coud you try it in this PR please?

Comment (mgencur) [reply ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.14.2.yaml:14]: Found out just today.

**Comment by mgencur** [comment]
/retest

(the respective PR on ci-tools side has been merged)

**Comment by alvaroaleman** [comment]
/test pj-rehearse
Now the PR got built as well :upside_down_face:

**Pull request merged**